odoo/addons/base/views/res_users_views.xml

```diff
@@ -200,13 +200,11 @@ if records:
                             <field name="email" invisible="1"/>
                             <label for="login" string="Email Address"/>
                             <h2>
-                                <field name="login"
-                                        placeholder="e.g. email@yourcompany.com"/>
+                                <field name="login" placeholder="e.g. email@yourcompany.com"/>
```

Comment:
@dasz-odoo 

This is purely stylistic change and not needed :innocent: 

Generally speaking, good rule of thumbs is that you don't make stylistic changes unless you also change the logic of the line. 
It makes diff hard to read. And also rewrites the history. If someone were to wonder about that line, git blame would be a bit harder to dig.

Obviously this is only one line but I am more talking for the future reference, please avoid purely stylistic changes.